hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson

```diff
@@ -3,15 +3,15 @@
 // SPDX-License-Identifier: Apache-2.0
 {
   name: "flash_ctrl"
-  // TODO: remove the common testplans if not applicable
   import_testplans: ["hw/dv/tools/dvsim/testplans/csr_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/mem_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/alert_test_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/intr_test_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/shadow_reg_errors_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/stress_all_with_reset_testplan.hjson",
-                     // TODO: Top-level specific Hjson imported here. This will likely be resolved
+                     // For the future development, follow comments can be considered. 
+                     // Top-level specific Hjson imported here. This will likely be resolved
```

Comment:
Do you mind using the ICEBOX[#(issue_number)] format, and maybe just tag the IPGen flow issue number?